Oqtane.Client/Modules/Admin/Sites/Add.razor

```diff
@@ -234,12 +222,19 @@ else
                 site.DefaultThemeType = themetype;
                 site.DefaultLayoutType = (layouttype == null ? "" : layouttype);
                 site.DefaultContainerType = containertype;
-                site = await SiteService.AddSiteAsync(site, aliases[0]);
+                
+                site = await SiteService.AddSiteAsync(site);
 
-                foreach(Alias alias in aliases)
+                List<Alias> aliases = new List<Alias>();
+                urls = urls.Replace("\n", ",");
+                foreach (string name in urls.Split(new char[] { ',' }, StringSplitOptions.RemoveEmptyEntries))
```

Comment:
Can you declare a separator as `static readonly` which hold the value `new char[] { ',' }`, and it will be nice if you are using the Split before the `foreach`